mysql-test/suite/galera/include/galera_wsrep_recover.inc

```diff
@@ -1,5 +1,14 @@
 --echo Performing --wsrep-recover ...
+#
+# When mysqld is run by a root user it will fail to start unless
+# we specify what user is ran it (using "--user root" option):
+#
+if ($MYSQL_TEST_ROOT == 1) {
+--exec $MYSQLD --defaults-group-suffix=.$galera_wsrep_recover_server_id --defaults-file=$MYSQLTEST_VARDIR/my.cnf --innodb --wsrep-recover --user root > $MYSQL_TMP_DIR/galera_wsrep_recover.log 2>&1
```

Comment:
https://mariadb.com/kb/en/library/running-mysqld-as-root/

Reply:
We can delete this "if", but then it will be impossible to run the mtr tests that use galera_wsrep_recover.inc under the root. I understand that this is not a standard mode, but without this "if" running mtr under the root user will lead to errors in several tests.

Reply:
From Elenst "i don't think we ever guaranteed that
mysqld itself barely runs with root user"

Reply:
Okay, I'll delete root user support.